content/zh/docs/releases/supported-releases/index.md

```diff
@@ -18,23 +18,23 @@ test: n/a
 - [控制面/数据面偏差](#control-planedata-plane-skew)
 - [Istio 版本的支持状态](#support-status-of-istio-releases)
 - [没有已知的 CVE 和常见漏洞的受支持版本](#supported-releases-without-known-common-vulnerabilities-and-exposures-cves)
-- [Istio 和 Envoy 之间的关系](#supported-envoy-versions)
+- [被支持的 Envoy 版本](#supported-envoy-versions)
 
 ## 支持策略 {#support-policy}
 
 每个提交会产生 Istio 构建版本。大约每个季度我们会构建一个次要版本并进行大量测试和发行认证。
 并对在次要版本中发现的问题发布补丁版本。
 
 不同类型代表不同的产品质量，Istio 团队对其有不同的支持力度。
-在这种情况下，标明 **支持** 意味着我们将为关键问题发布补丁并提供技术支持。
+在这种情况下，标明**支持**意味着我们将为关键问题发布补丁并提供技术支持。
 另外，第三方和合作伙伴也可能提供长期支持方案。
 
-|类型              | 支持级别                                                                                                         | 质量和建议使用场景
-|------------------|-----------------------------------------------------------------------------------------------------------------------|----------------------------
-|开发构建 | 不支持                                                                                                            | 危险，不完全可靠。建议用于实验、测试场景。
-|次要版本     | 在 N+2 次要版本发布后提供6个星期的支持（例如在 1.13.0 版本发布后，对 1.11 版本提供 6 个星期的支持
-|补丁             | 与相应的次要版本相同                                                                               | 建议用户在补丁可用时尽快采用。
-|安全补丁    | 与补丁相同，但是除了上一个补丁程序的安全修复之外，它将不包含任何其他代码 | 因为安全修复的重要性，**强烈** 建议用户采用发行版本后的安全补丁。
+| 类型     | 支持级别                                                                                       | 质量和建议使用场景                                              |
```

Comment:
Does this table only adjust spaces? It is difficult to identify the changes made.

Reply:
Yes, only formatter and add a missed `)`.

Reply:
okay, noted.